SingularityUI/app/components/tasks/Columns.jsx

```diff
@@ -0,0 +1,302 @@
+import React, { PropTypes } from 'react';
+import Column from '../common/table/Column';
+
+import Utils from '../../utils';
+
+import JSONButton from '../common/JSONButton';
+import Glyphicon from '../common/atomicDisplayItems/Glyphicon';
+
+export const TaskId = (
+  <Column
+    label="Task ID"
+    id="taskId"
+    key="taskId"
```

Comment:
`key` should be removed, as this gets handled within the UITable's code. 

(Although my handling uses the index when it should actually use the `id`.
That shouldn't matter unless you're moving columns around though.)